proto/viam/app/v1/robot.proto

```diff
@@ -256,12 +256,14 @@ message RemoteAuth {
 
 message AgentInfo {
```

Comment:
nit: I saw that we are making AgentInfo from optional -> required in the ConfigRequest. Should we make a note here to do that once we have the setup to always send that info in the request?

Reply:
I should have commented about this my bad. I tried this and its more complicated IMO, I'll tackled it after some other parts of the project are done. I wanted to move quicker than think of deprecation so I created a separate ticket for this yesterday: https://viam.atlassian.net/browse/APP-2142